src/buffer_ops/open_close.c

```diff
@@ -46,6 +46,8 @@ pmix_pointer_array_t pmix_bfrop_types = {{0}};
 pmix_data_type_t pmix_bfrop_num_reg_types = PMIX_UNDEF;
 static pmix_bfrop_buffer_type_t pmix_default_buf_type = PMIX_BFROP_BUFFER_NON_DESC;
 
+
+PMIX_EXPORT
```

Comment:
Yes, we like to use it... so we always make this change every time we pull down new bits.   Its just very convenience for the RM to be able to use this code to move PMIx-related structures around.